packages/components/psammead-radio-schedule/src/ProgramCard/index.jsx

```diff
@@ -0,0 +1,254 @@
+  ${({ script }) => script && getPica(script)};
+  color: ${({ headerTextColor }) => headerTextColor};
+  margin: 0; /* Reset */
+`;
+
+const HeadingContentWrapper = styled.span.attrs({ role: 'text' })``;
+
+const LabelWrapper = styled.span`
+  ${({ service }) => service && getSansBold(service)};
+  ${({ script }) => script && getPica(script)};
+  color: ${({ headerLabelColor }) => headerLabelColor};
+`;
+
+const TitleWrapper = styled.span`
+  color: ${({ titleColor }) => titleColor};
+  padding: ${GEL_SPACING} 0;
+  display: block;
+  ${({ service }) => service && getSansRegular(service)};
+  ${({ script }) => script && getPica(script)};
+`;
+
+const SummaryWrapper = styled.p`
+  ${({ service }) => service && getSansRegular(service)};
+  ${({ script }) => script && getLongPrimer(script)};
+  color: ${C_METAL};
+  padding-bottom: ${GEL_SPACING_DBL};
+  margin: 0; /* Reset */
+`;
+
+const ButtonWrapper = styled.div`
+  ${({ service }) => service && getSansRegular(service)};
+  ${({ script }) => script && getMinion(script)};
+  padding: ${GEL_SPACING};
+  background-color: ${({ backgroundColor }) => backgroundColor};
+  color: ${({ durationColor }) => durationColor};
+  border-top: 1px solid transparent;
+`;
+
+const IconWrapper = styled.span`
+  > svg {
+    color: ${({ durationColor }) => durationColor};
+    fill: currentColor;
+    width: 1.0625rem;
+    height: ${GEL_SPACING_DBL};
+    margin: 0;
+  }
+`;
+
+const DurationWrapper = styled.time`
+  ${({ dir }) =>
+    dir === 'ltr'
+      ? `padding-left: ${GEL_SPACING};`
+      : `padding-right: ${GEL_SPACING};`}
+`;
+
+const DurationTextWrapper = styled.span.attrs({ 'aria-hidden': 'true' })``;
+
+const programStateConfig = {
+  live: {
+    backgroundColor: C_POSTBOX,
+    headerLabelColor: C_POSTBOX,
+    headerTextColor: C_EBON,
+    titleColor: C_SHADOW,
+    durationColor: C_WHITE,
+  },
+  next: {
+    backgroundColor: C_WHITE,
+    headerLabelColor: C_KINGFISHER,
+    headerTextColor: C_METAL,
+    titleColor: C_METAL,
+    durationColor: C_KINGFISHER,
+  },
+  onDemand: {
+    backgroundColor: C_EBON,
+    headerTextColor: C_EBON,
+    titleColor: C_SHADOW,
+    durationColor: C_WHITE,
+  },
+};
+
+const renderHeaderContent = ({
+  state,
+  link,
+  stateLabel,
+  brandTitle,
+  episodeTitle,
+  service,
+  script,
+  startTime,
+}) => {
+  const isOnDemand = state === 'onDemand';
+  const isLive = state === 'live';
+  const hiddenTextProps = stateLabel === 'Live' ? { lang: 'en-GB' } : {};
+
+  const labelWrapperProps = isLive ? { 'aria-hidden': 'true' } : {};
+
+  const content = (
+    <HeadingContentWrapper>
+      {!isOnDemand && (
+        <>
+          <LabelWrapper
+            service={service}
+            script={script}
+            {...labelWrapperProps}
+            {...programStateConfig[state]}
+          >
+            {`${stateLabel.toUpperCase()}`}
+          </LabelWrapper>
+          <VisuallyHiddenText {...hiddenTextProps}>
```

Comment:
I think this is good to merge into the alpha as long as we raise tickets to:

- Create the timestamp correctly, using the same timestamp that is used in the start time indicator (will be easier to do when they're both merged in)
- Get requirements for duration formatting and refactor that in here based on what they are
- Make sure the visually hidden live label has the same logic in here, story promos, bulletins (and anywhere else it will be used). We can merge it in as is to the alpha and then pull it out and share it across each?

Reply:
https://github.com/bbc/psammead/issues/3036
https://github.com/bbc/psammead/issues/3037